subprojects/launcher/src/integTest/groovy/org/gradle/launcher/daemon/ProcessCrashHandlingIntegrationTest.groovy

```diff
@@ -254,7 +254,7 @@ class ProcessCrashHandlingIntegrationTest extends DaemonIntegrationSpec {
                 // We expect AttachConsole to fail with a particular error if the
                 // provided pid is not attached to a console
```

Comment:
⭕️ Maybe we should mention in the comment that the error depends on the OS version